### ISSUE TITLE
Fix Help dialogue

### ISSUE THREAD

**Issue by airon90** [samdroid-apps/reddit-is-gtk#46]
In Help dialogue you say that SFR has been build for KDE and with Qt5 but it is built for GNOME community and with GTK3.
See screenshot.

![schermata da 2016-09-05 14-58-48](https://cloud.githubusercontent.com/assets/139841/18248575/626dd670-7379-11e6-85b4-1daacb9321e3.png)

**Comment by samdroid-apps** [comment]
Yep, that is 100% a bug.

I forgot that when I built this app, I was actually using the Enlightenment
Foundation Libraries, which is said to be one of the best toolkits;
https://what.thedailywtf.com/topic/15001/enlightened

On a more serious note; the app was previously called "Reddit is Gtk+" so
the whole Qt thing was just a joke.  I had to rename it because I actually
went on to read the reddit branding guidelines.  I suppose the joke doesn't
make sense with the new name.
